holoviews/operation/normalization.py

```diff
@@ -175,3 +177,29 @@ def _normalize_raster(self, raster, key):
                 if range:
                     norm_raster.data[:,:,depth] /= range
         return norm_raster
+
+
+class normalize_group(Operation):
+    """
+    Group-wise min-max normalisation.
```

Comment:
Docstring should say more including that the range on the dimensions are being computed...